src/yuzu/configuration/configure_dialog.cpp

```diff
@@ -169,6 +170,8 @@ void ConfigureDialog::PopulateSelectionList() {
 
 void ConfigureDialog::OnLanguageChanged(const QString& locale) {
     emit LanguageChanged(locale);
+    //  Reloading the game list is needed to force retranslation.
+    UISettings::values.is_game_list_reload_pending.exchange(true);
```

Comment:
Since the return value from exchange isn't used, this should be able to be:

```suggestion
    UISettings::values.is_game_list_reload_pending = true;
```

Reply:
I was wondering about that. I looked up what exchange does and it apparently does a [read-modify-write](https://en.cppreference.com/w/cpp/atomic/atomic/exchange)

If you do a code search for `is_game_list_reload_pending` you'll find 4 other times where exchange is used, but the return value isn't used. 

Maybe there is some multithreaded related stuff happening under the hood?

https://github.com/yuzu-emu/yuzu/blob/46da380b571b46d85bf0a6e62ce7a74c9fdce832/src/yuzu/configuration/configure_per_game_addons.cpp#L66-L67

https://github.com/yuzu-emu/yuzu/blob/46da380b571b46d85bf0a6e62ce7a74c9fdce832/src/yuzu/configuration/configure_filesystem.cpp#L134-L139


https://github.com/yuzu-emu/yuzu/blob/46da380b571b46d85bf0a6e62ce7a74c9fdce832/src/yuzu/configuration/configure_general.cpp#L77-L89

https://github.com/yuzu-emu/yuzu/blob/46da380b571b46d85bf0a6e62ce7a74c9fdce832/src/yuzu/configuration/configure_ui.cpp#L123-L125

Reply:
It is fine to use an assignment here, since an `operator=` for an atomic will atomically set the value as well. `exchange` in all of these cases isn't required either and can just be assignments